src/sage/graphs/generic_graph.py

```diff
@@ -21014,6 +21016,95 @@ def layout_graphviz(self, dim=2, prog='dot', **options):
         positions = dot2tex.dot2tex(self.graphviz_string(**options), format='positions', prog=prog)
 
         return {key_to_vertex[key]: pos for key, pos in positions.items()}
+    
+    def layout_tutte(self, external_face, external_face_pos=None, **options):
+        """
+
+        Compute graph layout based on a Tutte embedding.
+
+        The graph must be 3-connected and planar.
+
+        INPUT:
+
+        - ``external_face`` -- list; the external face to be made a polygon
+
+        - ``external_face_pos`` -- list (default: ``None``); the positions of the vertices 
+          of the external face. If ``None``, will automatically generate a unit sided regular polygon.
```

Comment:
this line requires 2 more spaces indentation. See other methods for examples.

Reply:
In the other methods, it seems like the inputs have two spaces of indentation for an input element which requires more than a line of explanation. Is that not correct?